client/src/components/pages/personalProfile/style.css

```diff
@@ -0,0 +1,41 @@
+.form {
```

Comment:
@AbdallahAmmar96 This class name might need to be made more specific (for example .personalProfile-form) because .form is a pretty generic name and might already be used elsewhere (so styles could leak between this form and other forms)